.github/workflows/cae.yml

```diff
@@ -0,0 +1,65 @@
+name: CAE Build/Release
```

Comment:
since we are rebranding the CAE tool to IPT, can we rename this file to ipt.yml? and replace all occurrence of `cae` in this PR?

Reply:
I can do that if that's the direction we are moving toward with naming this app and project

Reply:
I rename CAE reference to IPT